ui/admin/app/components/scope-picker/index.hbs

```diff
@@ -0,0 +1,99 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<Hds::Dropdown
+  @enableCollisionDetection={{true}}
+  @matchToggleWidth={{true}}
+  {{! HDS provided class to automatically fade in/out }}
+  class='hds-side-nav-hide-when-minimized scope-picker'
+  as |D|
+>
+  {{#let this.currentScope as |scope|}}
+    <D.ToggleButton
+      @text={{scope.name}}
+      @icon={{scope.icon}}
+      class='scope-picker__toggle-button'
```

Comment:
None blocking comment: is this class need for the comment on line 9?

Reply:
this class is needed to add a css style to the button